modules/sms_blast/src/SmsBlastForm.php

```diff
@@ -42,21 +84,25 @@ public function buildForm(array $form, FormStateInterface $form_state) {
    * {@inheritdoc}
    */
   public function submitForm(array &$form, FormStateInterface $form_state) {
-    $result = db_select('sms_user', 'su')
-      ->fields('su', array('uid'))
-      ->condition('status', 2)
+    $sms_message = new SmsMessage();
+    $sms_message->setMessage($form_state->getValue('message'));
+
+    $ids = $this->phoneNumberVerificationStorage->getQuery()
+      ->condition('status', 1)
```

Comment:
We swapped systems, there is no value mapping. The phone number verifications are binary. Since we are doing a fancy SQL query it has to be 1 or 0, not TRUE/FALSE. When passing around verification objects we do ->setStatus(bool)